data/templates/dovecot/dovecot.conf

```diff
@@ -8,7 +8,11 @@ mail_home = /var/mail/%n
 mail_location = maildir:/var/mail/%n
 mail_uid = 500
 
+{% if pop3_enabled %}
+protocols = imap sieve pop3
```

Comment:
You can do a one line `protocols = imap sieve {% if pop3_enabled %}pop3{% endif %}` if you want instead